feature sequence

mrugacz95 opened the pull request.

mrugacz95 committed:
sequence

mrugacz95 committed:
Merge remote-tracking branch 'origin/master' into feature/001sequnce

KolorowyAleksander commented on src/main/java/io/riders/controllers/SequenceController.java:
```diff
@@ -0,0 +1,20 @@
+@Controller
+public class SequenceController {
+    @RequestMapping("/sequence")
+    public @ResponseBody SequenceModel sequence(@RequestParam(value = "length", required = false, defaultValue = "10") Integer length){
```
int

KolorowyAleksander commented on src/main/java/io/riders/models/SequenceModel.java:
```diff
@@ -0,0 +1,22 @@
+ */
+public class SequenceModel {
+    private List<Integer> sequence = new ArrayList<>();
+    public SequenceModel(Integer lenght){
```
int

KolorowyAleksander commented on src/main/resources/static/js/sequence.js:
```diff
@@ -0,0 +1,13 @@
+ * Created by mruga on 11.06.2017.
+ */
+$(function() {
+    $('#dice').on('click', function() {
```
Different id than '#dice'

mrugacz95 committed:
Fixes

mrugacz95 committed:
Fixes srixes

mrugacz95 committed:
sequence in navbar

mrugacz95 committed:
ja sequence

KolorowyAleksander committed:
Fix Marcin bad codes

Fix formatting in some classes
Fix indexOutOfBoundException
Remove required login from endpoint

mrugacz95 committed:
Lotto and sequence implementation

KolorowyAleksander commented:
:+1:

KolorowyAleksander commented:
:shipit:

mrugacz95 committed:
Error container

KolorowyAleksander commented:
:+1:
